exampleapp/BUILD

```diff
@@ -0,0 +1,13 @@
+android_binary(
+    name = "exampleapp",
+    srcs = glob(["src/main/java/com/google/keytransparency/exampleapp/*.java"]),    
```

Comment:
We prefer to not use glob in BUILD files if we can. 
Would you mind breaking these files out into a list?